xmtp_api_grpc/src/lib.rs

```diff
@@ -188,6 +188,8 @@ mod tests {
             .await
             .unwrap();
 
+        // Sleep 100ms to ensure subscription is updated
+        tokio::time::sleep(std::time::Duration::from_millis(100)).await;
```

Comment:
I noticed some flakiness in this test because of race conditions. This should resolve.